charts/infra-apps/examples/argocd.yaml

```diff
@@ -0,0 +1,78 @@
+_: &url "https://argocd.example.com"
+_: &host "argocd.example.com"
+
+namespaces:
+  namespaces:
+    infra-argocd:
+      additionalLabels:
+        k8s.adfinis.com/rbac-definition: "infra-rbac"
+
+argoconfig:
+  apps:
+    infra-argocd:
+      # anable deploying Argo CD
+      enabled: true
+      values:
+        server:
+          config:
+            # configure main Argo CD URL
+            url: *url
+            # add a dex.config to enable auth via an external IdP or LDAP server
+            #
+            # The example below uses and external LDAP server and needs adapting
+            # by changing the example.com endpoints as well as the searches if
+            # you are on a non AD environment.
+            dex.config: |
```

Comment:
Rewriting argoconfig to a library chart will make loading this from an external file/template possible.

Reply:
I'm reconsidering this since it would probably mean having more magic yaml merging than makes sense. Most likely I'll revisit this if we come across an example where the config is large enough so this makes more sense.